source/connect/advanced-connection-options.txt

```diff
@@ -0,0 +1,46 @@
+.. _advanced-connection-options:
+
+===========================
+Advanced Connection Options
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/intro-advanced-connection-options.rst
+
+Options
+-------
+
+:ref:`general-connection-tab`
+  Learn how to select a Connection String Scheme for your deployment. 
+
+:ref:`authentication-connection-tab`
+  Learn how to connect deployments that require authentication.
+
```

Comment:
(wording)
This pertains to each of these line items.

Are you learning how to connect deployments WITH TLS/SSL?

Or...
Are you learning how to connect deployments that have TLS/SSL enabled?

Reply:
Will double check in external review, but it looks like it should all be "WITH TLS/SSL".